lib/oeqa/selftest/cases/updater_qemux86_64.py

```diff
@@ -204,6 +207,26 @@ def test_provisioning(self):
         verifyProvisioned(self, machine)
 
 
+class DeviceCredProvTestsNonOSTree(DeviceCredProvTests):
+    def setUpLocal(self):
+        global UBOOT_ENABLE
+        UBOOT_ENABLE = 'no'
+        self.append_config('DISTRO = "poky"')
+        self.append_config('DISTRO_FEATURES_append = " systemd"')
+        self.append_config('VIRTUAL-RUNTIME_init_manager = "systemd"')
+        self.append_config('PREFERRED_RPROVIDER_virtual/network-configuration ??= "networkd-dhcp-conf"')
+        self.append_config('PACKAGECONFIG_pn-aktualizr = ""')
+        self.append_config('IMAGE_INSTALL_append += "aktualizr"')
+        self.append_config('IMAGE_INSTALL_remove += " aktualizr-shared-prov"')
+        self.append_config('IMAGE_INSTALL_append += " aktualizr-device-prov"')
```

Comment:
For the ease of testing, it might be better to use shared credential provisioning here.

But either way, I like the idea of reusing one of the existing tests and extending it like this.

Reply:
So would you prefer to have both of those tests or only shared provisioning?

Reply:
Just shared credential provisioning is enough, I think.